.buildkite/pipelines/check_pe_area.sh

```diff
@@ -19,7 +19,7 @@ echo $pe_dir
 
 # "echo" to unglob why not
 resdir=`echo $pe_dir/*synthesis/results_syn`
-egrep ^Tile_PE $resdir/final_area.rpt | awk -v max_area=11000 '
+egrep ^Tile_PE $resdir/final_area.rpt | awk -v max_area=8500 '
```

Comment:
similarly, this script is only invoked for (currently working) TSMC buildkite tests, and is not used for GF tests